BassBotDotNET/Program.cs

```diff
@@ -32,12 +41,21 @@ public async Task MainAsync()
         await Task.Delay(-1);
     }
 
+    /// <summary>
+    /// Basic message logger.
+    /// </summary>
+    /// <param name="msg"></param>
+    /// <returns></returns>
```

Comment:
I noticed that you don't write the type of return variable in the C# doc.

Reply:
Thanks for catcching this, this is a mistake LOL

Reply:
I apparently missed the <param> comment as well omfg. I think I was rushing while making the docs LOL

Reply:
AS WELL AS ALOT OF OTHER PARAM AND RETURN DOCS my god